src/components/CurrencyInputPanel/FiatValue.tsx

```diff
@@ -25,18 +26,24 @@ export function FiatValue({
   }, [priceImpact, theme.green1, theme.red1, theme.text3, theme.yellow1])
 
   return (
-    <TYPE.body fontSize={14} color={fiatValue ? theme.text2 : theme.text4}>
+    <TYPE.body fontSize={14} color={fiatValue ? theme.text3 : theme.text4} fontWeight={500}>
       {fiatValue ? (
         <Trans>
-          ~$ <HoverInlineText text={fiatValue?.toSignificant(6, { groupSeparator: ',' })} />
+          $
+          <HoverInlineText
+            text={fiatValue?.toSignificant(6, { groupSeparator: ',' })}
+            textColor={fiatValue ? theme.text3 : theme.text4}
+          />
         </Trans>
       ) : (
         ''
       )}
       {priceImpact ? (
         <span style={{ color: priceImpactColor }}>
           {' '}
-          (<Trans>{priceImpact.multiply(-1).toSignificant(3)}%</Trans>)
+          <MouseoverTooltip text={'The estimated difference between the USD values of input and output amounts.'}>
```

Comment:
don't forget to translate, can use `t` from `const { t } = useTranslation()` I think

Reply:
Looks like we dont have this hook anymore?

Reply:
hm so ideally you could use `<Trans>` here. However mouseovertooltip requires a `string` (i'm assuming, didn't actually check) so you have to use `t`